developmental-dynamics.csl

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" initialize-with-hyphen="false" default-locale="en-US">
+  <info>
+    <title>Developmental Dynamics</title>
+    <title-short>Dev Dyn</title-short>
+    <id>http://www.zotero.org/styles/developmental-dynamics</id>
+    <link href="http://www.zotero.org/styles/developmental-dynamics" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
+    <link href="https://https://anatomypubs.onlinelibrary.wiley.com/hub/journal/10970177/homepage/forauthors.html#refs" rel="documentation"/>
+    <author>
+      <name>Engin Ozkan</name>
+      <email>enozkan78@gmail.com</email>
+    </author>
+    <contributor>
```

Comment:
remove all the contributors. You're the author and the creators of AMA are acknowledged via the template link.